packages/core-components/src/lib/atoms/inputs/button-group/ButtonGroup.svelte

```diff
@@ -43,30 +46,32 @@
 	));
 </script>
 
-<div class="inline-flex flex-col mt-2 mb-4 mx-1">
-	{#if title}
-		<span class="text-gray-500 block mb-1">{title}</span>
-	{/if}
-	<div class="inline-flex" role="group">
-		{#if preset}
-			{#if presets[preset]}
-				{#each presets[preset] as { value, valueLabel }}
-					<ButtonGroupItem {value} {valueLabel} />
-				{/each}
-			{:else}
-				<span class="text-red-500 font-bold text-sm">{preset} is not a valid preset</span>
-			{/if}
-		{:else}
-			<slot />
-			{#if hasQuery}
-				{#if $query.error}
-					{$query.error}
-				{:else}
-					{#each $query as { label, value }}
-						<ButtonGroupItem {value} valueLabel={label} />
+<HiddenInPrint enabled={hideDuringPrint}>
+	<div class="inline-flex flex-col mt-2 mb-4 mx-1">
```

Comment:
might be out of scope but we are not super visually consistent with new dropdown any more
![CleanShot 2024-02-06 at 18 29 30@2x](https://github.com/evidence-dev/evidence/assets/58074498/75dae84f-17ff-4c35-a0da-63d0183ba05c)

Reply:
I can clean this up